### PR TITLE
chore: some typos were fixed in the README.md file

### PR DESCRIPTION
**Pull request by brunosc10699**
### Related Issues
I would like to contribute to this project by also suggesting some typos fixes in the README.md file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ Please read the [code] so that you can understand what actions will and will not
 
 ### Good First Issues
 
-We have a list of [good first issues][gfi] that contain bugs which have a relatively limited scope. This is a great place to get started, gain experience, and get familiar with our contribution process.
+We have a list, of [good first issues][gfi], that contains bugs, which have a relatively limited scope. This is a great place to get started, gain experience, and get familiar with our contribution process.
```
Comment (igorgiumellizup) [inline README.md:78]: I think the ```of [good first issues][gfi]```part doesn't need to be between comas

Comment (brunosc10699) [reply README.md:78]: Cool Igor! I'm just a newbie to Git and I need your help on how to go about modifying my PR. After I remove the commas should I make a new PR? That's it?

Comment (igorgiumellizup) [reply README.md:78]: Send me a private msg on G chat

**Review by brunosc10699 — COMMENTED** [review]
Removed excess commas

File: README.md (hunk 1 above)
Comment (brunosc10699) [reply README.md]: I'm having trouble sending you a private message on G chat. Could you send one for me?

**Comment by uziasferreirazup** [comment]
Hello I appreciate your contribution, but it's necessary to assign off your commit to be accept in the project.
it's very simple you can read this page to know how to do this: https://github.com/ZupIT/beagle/blob/main/doc/contributing/dco_rules.md

**Review by uziasferreirazup — APPROVED** [review]

**Comment by uziasferreirazup** [comment]
I will merge this PR because we are going to use squash and merge, but next time it will be necessary to signoff all commits:
the command to use in the next PR to help you sign off all commits, like in this PR has 3 commit and one merge commit, the merge commit don't account `git rebase --signoff @~3` the number 3 can be change to total commit do you need to signoff

**Comment by brunosc10699** [comment]
Thank you @uziasferreirazup! It's a very specific command to me. I was searching on the internet how to edit an old commit and I only found incomplete information till now, and I was afraid of messing your job.

**Review by Tiagoperes — APPROVED** [review]

**Pull request merged**